code/modules/hydroponics/seed.dm

```diff
@@ -726,33 +726,35 @@
 				total_yield = max(1,total_yield)
 
 		. = list()
-		for(var/i = 0;i<total_yield;i++)
-			var/obj/item/product = new product_type(get_turf(user),name)
-			. += product
+		if(ispath(product_type, /obj/item/stack))
+			. += new product_type(get_turf(user), total_yield)
```

Comment:
does this properly handle splitting stacks when `total_yield` is greater than the max stack size?